qiskit/transpiler/passes/synthesis/hls_plugins.py

```diff
@@ -559,6 +559,7 @@
     synth_mcx_1_clean_b95,
     synth_mcx_gray_code,
     synth_mcx_noaux_v24,
+    synth_mcx_noaux_hp24,
```

Comment:
you should add it to the toctree, as currently it does not appear in the docs

Reply:
Done in [97553c8](https://github.com/Qiskit/qiskit/pull/14666/commits/97553c812a746c6eda4ec38fdc3d6e19052e3197), [f6e2c6e](https://github.com/Qiskit/qiskit/pull/14666/commits/f6e2c6e33530d3dfabaf09dbcee3f7e724e6690a)